.pipelines/pipeline.yaml

```diff
@@ -42,26 +42,17 @@ stages:
           name: "$(BUILD_POOL_NAME_DEFAULT)"
         steps:
           - script: |
-              GOOS=windows make all-binaries VERSION=$(TAG)
-            name: "BuildWindows"
-            displayName: "Build Windows"
-
-          - script: |
-              GOOS=linux make all-binaries VERSION=$(TAG)
-            name: "BuildLinux"
-            displayName: "Build Linux"
+              make all-binaries-platforms GOOS=linux,windows GOARCH=amd64,arm64 VERSION=$(TAG)
```

Comment:
drop the GOOS and GOARCH so that we use the Makefile defaults in the pipeline runs